client/src/modules/stock/depot-selection.modal.html

```diff
@@ -0,0 +1,72 @@
+<div class="modal-header" data-depot-selection-modal>
+  <ol class="headercrumb">
+    <li class="static" translate>TREE.DEPOTS</li>
+    <li class="title" translate>DEPOT.MODAL.SELECTION</li>
+  </ol>
+</div>
+
+<div class="modal-body">
+
+  <!-- link to depots page if depots not found -->
+  <p class="text-info">
+    <i class="fa fa-question-circle-o"></i>
+    <span translate>DEPOT.MODAL.NO_DEPOTS</span>
+
+    <a href ui-sref="depots.create">
+      <i class="fa fa-link"></i> <span translate>DEPOT.ADD_DEPOT</span>
+    </a>
+  </p>
+
+  <div uib-alert class="alert-warning" ng-if="$ctrl.hasNoDefaultDepot">
+    <p>
+      <strong> <i class="fa fa-warning"></i> <span translate>DEPOT.MODAL.NO_DEFAULT_DEPOT</span></strong>
+      <br />
+      <span translate>DEPOT.MODAL.SELECTION_EXPLANATION</span>
+      <a href ui-sref="depots.create"><b><span translate>DEPOT.ADD_DEPOT</span></b></a>.
```

Comment:
It would be nice if this was a colored yellow like the rest of the alert.  It is currently blue, which really stands out and clashes with the rest of the colors.